testing/integration_test.go

```diff
@@ -27,7 +27,7 @@ import (
 	"text/template"
 	"time"
 
-	"github.com/GoogleCloudPlatform/google-cloud-go/profiler/proftest"
+	"cloud.google.com/go/profiler/proftest"
```

Comment:
This change is not really related to the fix, but using consistent path with https://code.googlesource.com/gocloud/+/master/profiler/integration_test.go#31.